documentation/disaster-recovery.md

```diff
@@ -103,25 +107,25 @@ In case the database instance is lost, the objectives are:
 
 ### Recreate the lost postgres database instance
 
-In case the database service is deleted or in an inconsistent state we must recreate it and repopulate it.
+Please note, this process could take about circa 60 min to compleate. In case the database service is deleted or in an inconsistent state we must recreate it and repopulate it.
 First make sure it is fully gone by running cf services and cf delete-service if necessary.
-Then recreate the lost postgres database instance using the makefile recipes `terraform-app-plan` and `terraform-app-apply`:
+Then recreate the lost postgress database instance using the following make recipes `terraform-app-plan` and `terraform-app-apply`:
 
 ```
-make <env> terraform-app-plan passcode=MyPasscode tag=main-08406f04dd9eadb7df6fcda5213be880d7df37ed-20201022090714 ## to see the proposed changes.
+[ aws-vault exec Deployments -- ] make <env> terraform-app-plan passcode=MyPasscode tag=main-08406f04dd9eadb7df6fcda5213be880d7df37ed-20201022090714 ## to see the proposed changes.
 ```
 
 ```
-make <env> terraform-app-apply passcode=MyPasscode tag=main-08406f04dd9eadb7df6fcda5213be880d7df37ed-20201022090714 ## apply proposed changes i.e. create new database instance.
+[ aws-vault exec Deployments -- ] make <env> terraform-app-apply passcode=MyPasscode CONFIRM_RESTORE=Yes tag=main-08406f04dd9eadb7df6fcda5213be880d7df37ed-20201022090714 ## apply proposed changes i.e. create new database instance.
```

Comment:
Isn't it `CONFIRM_RESTORE=YES` ? According to https://github.com/DFE-Digital/teaching-vacancies/blob/main/Makefile#L193